aiida/tools/importexport/dbexport/__init__.py

```diff
@@ -107,6 +107,7 @@ def export(
     allowed_licenses: Optional[Union[list, Callable]] = None,
     forbidden_licenses: Optional[Union[list, Callable]] = None,
     writer_init: Optional[Dict[str, Any]] = None,
+    batch_size: int = 100,
```

Comment:
Is 100 a reasonable default?
This needs performance testing.

Reply:
No idea and indeed would be best to performance test, but this is already the default value for `QueryBuilder.iterall` so it is not actually changing anything from the original code